### PR TITLE
Deploy fix

### LINKED ISSUE #376: MetaMask asks user to sign auth token
The auth token should be signed by the signing key (in the background without the user noticing it) and not by the Ethereum account key

### LINKED ISSUE #377: Backend crashes when sending a message
```
{"level":"info","message":{"account":"0xfCC50979560Ad027EE50D6545dBC847B4EC357D4","contact":"0x8756707634FAf6A8545cd9d1bd3DD36bD9aBCA73","method":"WS PENDING MESSAGE"}}
{"level":"warn","message":{"error":{},"method":"WS PENDING MESSAGE"}}
/Users/heikoburkhardt/repos/ens-mail/packages/backend/dist/messaging.js:167
                        callback({ error: "Can't add pending message" });
                        ^

TypeError: callback is not a function
    at /Users/heikoburkhardt/repos/ens-mail/packages/backend/dist/messaging.js:167:25
    at step (/Users/heikoburkhardt/repos/ens-mail/packages/backend/dist/messaging.js:52:23)
    at Object.next (/Users/heikoburkhardt/repos/ens-mail/packages/backend/dist/messaging.js:33:53)
    at fulfilled (/Users/heikoburkhardt/repos/ens-mail/packages/backend/dist/messaging.js:24:58)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

### LINKED ISSUE #378: Page is crashed on own account info click
```
Uncaught TypeError: Cannot read properties of undefined (reading 'account')
    at RightHeader (index.js:16237:1)
    at renderWithHooks (react-dom.development.js:16305:1)
    at mountIndeterminateComponent (react-dom.development.js:20074:1)
    at beginWork (react-dom.development.js:21587:1)
    at beginWork$1 (react-dom.development.js:27426:1)
    at performUnitOfWork (react-dom.development.js:26557:1)
    at workLoopSync (react-dom.development.js:26466:1)
    at renderRootSync (react-dom.development.js:26434:1)
    at recoverFromConcurrentError (react-dom.development.js:25850:1)
    at performConcurrentWorkOnRoot (react-dom.development.js:25750:1)
```

### PR DESCRIPTION
**Pull request by hai-ko**
closes #378
closes #377
closes #376

### CODE REVIEW

File: packages/backend/src/messaging.test.ts [hunk 1]
```diff
@@ -242,7 +242,8 @@ describe('Messaging', () => {
     });
 
     describe('pendingMessage', () => {
-        it('returns error if schema is invalid', async () => {
+        // TODO: enable after lib sends callback with pendingMessage
```
Comment (hai-ko) [inline packages/backend/src/messaging.test.ts:245]: should be fixed in #380

File: packages/backend/src/messaging.ts [hunk 2]
```diff
@@ -101,7 +101,7 @@ export function onConnection(app: express.Application & WithLocals) {
                     error,
                 });
 
-                return callback({ error });
+                //return callback({ error });
```
Comment (hai-ko) [inline packages/backend/src/messaging.ts:104]: should be fixed in #380

File: packages/backend/src/messaging.ts [hunk 3]
```diff
@@ -129,13 +129,14 @@ export function onConnection(app: express.Application & WithLocals) {
 
                 await addPending(account, contact, app.locals.redisClient);
 
-                callback({ response: 'success' });
+                // callback({ response: 'success' });
```
Comment (hai-ko) [inline packages/backend/src/messaging.ts:132]: should be fixed in #380

File: packages/backend/src/messaging.ts [hunk 4]
```diff
@@ -129,13 +129,14 @@ export function onConnection(app: express.Application & WithLocals) {
...
             } catch (error) {
                 app.locals.logger.warn({
                     method: 'WS PENDING MESSAGE',
-                    error,
+                    error: (error as Error).toString(),
                 });
-                callback({ error: "Can't add pending message" });
+
+                //return callback({ error: "Can't add pending message" });
```
Comment (hai-ko) [inline packages/backend/src/messaging.ts:139]: should be fixed in #380

**Review by AlexNi245 — APPROVED** [review]

**Pull request merged**